extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -483,4 +489,94 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
     const publisherOrigin = getSourceOrigin(this.getAmpDoc().getUrl());
     return publisherOrigin.replace(/https?:\/\//, '');
   }
+
```

Comment:
Wdyt about moving these to a `amp-story-form.js` file?

In terms of structure it could look like this file: https://github.com/ampproject/amphtml/blob/main/extensions/amp-story/1.0/semantic-render.js

It'd make any amp-form related code easier to re-use. Also, it's not specific to amp-story-page-attachment at all.

Reply:
Updated the PR with a new `amp-story-form.js` file